src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="col-sm-10 typeahead-dropdown pull-left">
+    <div *ngIf="!searchValue" (click)="open()" class="pointer details-dropdown" [ngClass]="getInitialValue().cssLabelClass?getInitialValue().cssLabelClass:''">{{ getInitialValue().value }}</div>
+    <div style="display: flex;">
```

Comment:
two things:
1. inline styles shouldn't be used
2. is it possible to do this without flex? flex will be removed post initial release